lib/components/Alert/Alert.tsx

```diff
@@ -61,9 +62,10 @@ export interface AlertProps extends React.Props<AlertComponentType> {
 export const Alert: React.StatelessComponent<AlertProps> = (props: AlertProps) => {
     const className = css({
         'alert-container': true,
-        'alert-info': props.type === AlertType.Information,
+        'alert-info': props.type == null || props.type === AlertType.Information,
```

Comment:
=== or ==?

Reply:
it is intended to be `==` basically if `props.type` is `null` or `undefined` it should default to `Information`

Reply:
sounds good!